packages/cloud-cognitive/src/global/styles/_mixins.scss

```diff
@@ -5,8 +5,13 @@
 // LICENSE file in the root directory of this source tree.
 //
 
-@mixin block-wrap($block-class) {
-  .#{$block-class} {
+@mixin block-wrap($block-class, $deep: 1) {
+  $selector: '';
+
+  @for $i from 1 through $deep {
+    $selector: $selector + '.' + $block-class;
+  }
+  #{$selector} {
```

Comment:
I don't think I like this. Well, I quite like it :-) but I'm not sure we should do it. It encourages the wrapping of great swathes of CSS in multi-class specificity wrappers that may be over the top and will bloat the CSS because these repeated classes will appear on every single CSS rule generated.

We could make this more efficient. For example,
```
@mixin block-wrap($block-class, $deep: 1) {
  $selector: $block-class;

  @for $i from 2 through $deep {
    $selector: $selector + ':not(\9)';
  }
  #{$selector} {
    @content;
  }
}
```
will I think achieve the same specificity with rather less bloat. However, I think it's actually better not to do this at all -- we should include the specificity boost on each rule that needs it, because that way we don't apply more specificity-busting than we need, and it is clear in the SCSS source which rules needed the specificity fixes.